lib/sycamore/sycamore/transforms/text_extraction/pdf_miner.py

```diff
@@ -73,31 +73,23 @@ def extract_document(self, filename: str, hash_key: str, use_cache=False, **kwar
             return pages
 
     @timetrace("PdfMinerPageEx")
-    def extract_page(
-        self, page: Optional[Union["PDFPage", "Image"]] = None, page_data: list[tuple[Any, Any, Any, Any]] = []
-    ) -> list[Element]:
+    def extract_page(self, page: Optional[Union["PDFPage", "Image"]] = None) -> list[Element]:
```

Comment:
So, I guess `page_data` wasn't needed after all?

Reply:
Not exactly. The functionality to support it was moved into a `TextExtractor` function.